src/mongo/grammar/mongoParser.ts

```diff
@@ -1022,6 +1023,7 @@ export class CollectionContext extends ParserRuleContext {
 
 export class FunctionCallContext extends ParserRuleContext {
 	public _FUNCTION_NAME: Token;
+	// tslint:disable-next-line:no-banned-terms
 	public arguments(): ArgumentsContext {
```

Comment:
Is there a list of banned terms I can look at?

Reply:
This is the best I can find: https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle

You could maybe call this a false positive since the scope is not global.

However, I see now this is a generated file, so I'll exclude it.